### PR TITLE
docs(tutorial): add reset button

### LINKED ISSUE #5860: [✨] add reset button to tutorial
### Is your feature request related to a problem?

As requested at https://discord.com/channels/842438759945601056/1207764738277122128

When loading the solution, add a reset button that allows the user to try it themselves

### Describe the solution you'd like

See above

### Describe alternatives you've considered

None

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ayush3160**
# Overview
Added Reset button to tutorial

https://www.loom.com/share/bb4a5b136c434541aa6409ec60e25e61?sid=e36ab391-9630-40d2-a5a6-eaeded3e4a93

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

fixes #5860 

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/tutorial/tutorial-content-footer.tsx [hunk 1]
```diff
@@ -1,20 +1,37 @@
-import { component$ } from '@builder.io/qwik';
+import { component$, useSignal } from '@builder.io/qwik';
 import { ensureDefaultFiles, type TutorialStore } from './layout';
 
 export const TutorialContentFooter = component$(({ store }: TutorialContentFooterProps) => {
+  let solutionViewState = useSignal(false);
```
Comment (gioboa) [inline packages/docs/src/routes/tutorial/tutorial-content-footer.tsx:5]:
```suggestion
  let solutionViewSig = useSignal(false);
```
I think it's more readable like this

**Review by gioboa — CHANGES_REQUESTED** [review]
it looks great :clap: 
I only added a small suggestion.
Thanks @ayush3160

File: packages/docs/src/routes/tutorial/tutorial-content-footer.tsx (hunk 1 above)
Comment (ayush3160) [reply packages/docs/src/routes/tutorial/tutorial-content-footer.tsx]: Done

**Comment by ayush3160** [comment]
@gioboa Thanks for the suggestion , I have changed the state name.

**Review by gioboa — APPROVED** [review]
Great 👏 @ayush3160

**Pull request merged**